DQMOffline/Configuration/scripts/cmsswSequenceInfo.py

```diff
@@ -39,7 +39,7 @@ def inspectsequence(seq):
     if seq.step not in ("HARVESTING", "ALCAHARVEST"):
         # cmsDriver refuses to run some steps on their own, so we add this to them
         # only loading a single module that we can blacklist later
-        otherstep = "RAW2DIGI:siPixelDigis,"
+        otherstep = ""
```

Comment:
what's the purpose of otherstep now?

Reply:
There's no need, you are right, so I will remove it from the file to avoid misunderstandings. Thanks!